### ISSUE TITLE
Unhandled Exception: Irondash init structure size differs between rust and dart

### ISSUE THREAD

**Issue by thinhpq0112** [superlistapp/super_native_extensions#131]
I copy code and launch the example super_clipboard successfully but when I click any button, all showed an error

```
E/flutter ( 6830): [ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: Irondash init structure size differs between rust and dart.
E/flutter ( 6830): #0      NativeFunctions.get (package:irondash_message_channel/src/native_functions.dart:100:9)
E/flutter ( 6830): #1      MessageChannelContext.forInitFunction (package:irondash_message_channel/src/message_channel.dart:85:25)
E/flutter ( 6830): #2      _getNativeContext (package:super_native_extensions/src/native/context.dart:16:34)
E/flutter ( 6830): #3      _nativeContext (package:super_native_extensions/src/native/context.dart:20:24)
E/flutter ( 6830): #4      _nativeContext (package:super_native_extensions/src/native/context.dart)
E/flutter ( 6830): #5      superNativeExtensionsContext (package:super_native_extensions/src/native/context.dart:30:25)
E/flutter ( 6830): #6      new ClipboardReaderImpl (package:super_native_extensions/src/native/clipboard_reader.dart:18:16)
E/flutter ( 6830): #7      ClipboardReader.instance (package:super_native_extensions/src/clipboard_reader.dart:6:43)
E/flutter ( 6830): #8      ClipboardReader.instance (package:super_native_extensions/src/clipboard_reader.dart)
E/flutter ( 6830): #9      ClipboardReader.readClipboard (package:super_clipboard/src/reader.dart:161:46)
E/flutter ( 6830): #10     _MyHomePageState._paste (package:yedda_notifications_app_flutter/pages/test_page.dart:202:42)
E/flutter ( 6830): #11     _InkResponseState.handleTap (package:flutter/src/material/ink_well.dart:1096:21)
E/flutter ( 6830): #12     GestureRecognizer.invokeCallback (package:flutter/src/gestures/recognizer.dart:253:24)
E/flutter ( 6830): #13     TapGestureRecognizer.handleTapUp (package:flutter/src/gestures/tap.dart:627:11)
E/flutter ( 6830): #14     BaseTapGestureRecognizer._checkUp (package:flutter/src/gestures/tap.dart:306:5)
E/flutter ( 6830): #15     BaseTapGestureRecognizer.handlePrimaryPointer (package:flutter/src/gestures/tap.dart:239:7)
E/flutter ( 6830): #16     PrimaryPointerGestureRecognizer.handleEvent (package:flutter/src/gestures/recognizer.dart:615:9)
E/flutter ( 6830): #17     PointerRouter._dispatch (package:flutter/src/gestures/pointer_router.dart:98:12)
E/flutter ( 6830): #18     PointerRouter._dispatchEventToRoutes.<anonymous closure> (package:flutter/src/gestures/pointer_router.dart:143:9)
E/flutter ( 6830): #19     _LinkedHashMapMixin.forEach (dart:collection-patch/compact_hash.dart:625:13)
E/flutter ( 6830): #20     PointerRouter._dispatchEventToRoutes (package:flutter/src/gestures/pointer_router.dart:141:18)
E/flutter ( 6830): #21     PointerRouter.route (package:flutter/src/gestures/pointer_router.dart:127:7)
E/flutter ( 6830): #22     GestureBinding.handleEvent (package:flutter/src/gestures/binding.dart:460:19)
E/flutter ( 6830): #23     GestureBinding.dispatchEvent (package:flutter/src/gestures/binding.dart:440:22)
E/flutter ( 6830): #24     RendererBinding.dispatchEvent (package:flutter/src/rendering/binding.dart:336:11)
E/flutter ( 6830): #25     GestureBinding._handlePointerEventImmediately (package:flutter/src/gestures/binding.dart:395:7)
E/flutter ( 6830): #26     GestureBinding.handlePointerEvent (package:flutter/src/gestures/binding.dart:357:5)
E/flutter ( 6830): #27     GestureBinding._flushPointerEventQueue (package:flutter/src/gestures/binding.dart:314:7)
E/flutter ( 6830): #28     GestureBinding._handlePointerDataPacket (package:flutter/src/gestures/binding.dart:295:7)
E/flutter ( 6830): #29     _invoke1 (dart:ui/hooks.dart:164:13)
E/flutter ( 6830): #30     PlatformDispatcher._dispatchPointerDataPacket (dart:ui/platform_dispatcher.dart:361:7)
E/flutter ( 6830): #31     _dispatchPointerDataPacket (dart:ui/hooks.dart:91:31)
E/flutter ( 6830): 
```

Version:

$ flutter --version
Flutter 3.7.12 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 4d9e56e694 (6 weeks ago) • 2023-04-17 21:47:46 -0400
Engine • revision 1a65d409c7
Tools • Dart 2.19.6 • DevTools 2.20.1

$ rustc --version
rustc 1.69.0 (84c898d65 2023-04-16)

android.ndkVersion "25.2.9519653"

**Comment by knopp** [comment]
I've seen similar reports but I'm unable to reproduce this. Can you please confirm whether this persists after 
1. Restarting the device
2. Flutter Clean

(preferably test it in this order).

**Comment by thinhpq0112** [comment]
> I've seen similar reports but I'm unable to reproduce this. Can you please confirm whether this persists after
> 
> 1. Restarting the device
> 2. Flutter Clean
> 
> (preferably test it in this order).

Oh, I tested but it still persists

**Comment by knopp** [comment]
Actually, the struct size has not changed in past 7 months. I'm wondering if this could maybe be a FFI bug where the struct is sized wrong. Does it happen for you with latest stable Flutter? I tried this with on several devices + simulator but I can't seem to reproduce the issue.

**Comment by knopp** [comment]
The problem was size struct having wrong type on 32bit platforms. This will be fixed in 0.8.1.
